### ISSUE TITLE
`splash dir` is not working anymore

### ISSUE THREAD

**Issue by toby-daigle** [splash-cli/splash-cli#44]
#### Basic informations
- **Splash CLI** version: `3.3.4-2`
- Node version: `10.15.3`
- Your OS: **macOS Mojave 10.14.3**

#### Before you go
- [x] I've read the [FAQ](https://github.com/splash-cli/splash-cli/blob/master/FAQ.md) section.
- [x] I've the latest version of **Splash CLI installed**
- [x] I've searched other issues like mine but I haven't found a solution.

# The ISSUE
This commit https://github.com/splash-cli/splash-cli/commit/a67d1357de8d39acd6405f1a0f2d1d0982e10faa#diff-1757314a518f2a5dba612e6c9f12b618 seems to have instroduced breaking changes.

You now import `pathFixer` instead of `fixPath` from `extra/utils`, but looks like the function name in `extra/utils` is still `pathFixer`

**To Reproduce**
1. Run `splash dir get`

**Expected behavior**
I expect to see the path to my splash images folder.

**Console Output**
```
TypeError: (0 , _utils.fixPath) is not a function
    at Function.getPath (/Users/tdaigle/.config/yarn/global/node_modules/splash-cli/build/commands/libs/Directory.js:158:30)
    at Object._callee$ (/Users/tdaigle/.config/yarn/global/node_modules/splash-cli/build/commands/dir.js:61:121)
    at tryCatch (/Users/tdaigle/.config/yarn/global/node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/Users/tdaigle/.config/yarn/global/node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/tdaigle/.config/yarn/global/node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:114:21)
    at step (/Users/tdaigle/.config/yarn/global/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
    at /Users/tdaigle/.config/yarn/global/node_modules/babel-runtime/helpers/asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (/Users/tdaigle/.config/yarn/global/node_modules/core-js/library/modules/_export.js:36:28)
    at Object.<anonymous> (/Users/tdaigle/.config/yarn/global/node_modules/babel-runtime/helpers/asyncToGenerator.js:14:12)

```

**Comment by toby-daigle** [comment]
Fix in progress

**Comment by Rawnly** [comment]
Thank you @toby-daigle !

**Comment by Rawnly** [comment]
Is this fixed in 3.3.5?

**Comment by toby-daigle** [comment]
YES! 3.3.5 is good!
